src/Makefile.am

```diff
@@ -92,6 +92,7 @@ endif
 BITCOIN_CORE_H = \
   addrdb.h \
   addrman.h \
+  attributes.h \
```

Comment:
Wrong file, should be `nodiscard.h`?

Reply:
`attributes.h` is correct but that file got lost in the last rebase. Now re-added :-)